tcmur_device.c

```diff
@@ -489,3 +542,20 @@ void *tcmur_dev_get_private(struct tcmu_device *dev)
 
 	return rdev->hm_private;
 }
+
+void tcmu_notify_cmd_timed_out(struct tcmu_device *dev, bool can_access_dev)
```

Comment:
Nit: is the `can_access_dev` needed? It looks like everyone calls this method w/ `true`

Reply:
Removed.